integration/runner/runner.go

```diff
@@ -68,6 +69,14 @@ func getAttributes(ipAddress, portStr string) (*cadvisorApi.Attributes, error) {
 	return &attributes, nil
 }
 
+func getIp(host string) (string, error) {
+	if *sshConfig == "" {
+		return host, nil
+	}
+	out, err := exec.Command("awk", fmt.Sprintf("/Host %s/ {getline; print $2}", host), *sshConfig).CombinedOutput()
```

Comment:
Fixed this by just requiring the hostname to resolve.  This works for gce and I think is an ok assumption to make.